packages/eslint-config-godaddy-minimal/index.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  extends: [
+    require('./extends')('eslint-config-godaddy-react-flow')
+  ],
+  rules: {
+    'complexity': 0,
+    'consistent-return': 0,
+    'id-length': 0,
+    'indent': [2, 2, {
+      VariableDeclarator: { var: 2, let: 2, const: 3 },
+      flatTernaryExpressions: true,
+      MemberExpression: 'off'
+    }],
+    'max-params': 0,
+    'max-nested-callbacks': 0,
+    'max-statements': 0,
```

Comment:
Like the rest of these rules, this one forces you to pause and reconsider your function. Massive "god functions" are pretty much always bad and should be broken down into smaller pieces, so I'd vote to keep this.

Reply:
I agree that they're bad; that's why they should be discussed in code review. If someone doesn't realize that they're writing a function that's too long, that's a good sign that they should get assistance refactoring it as part of a code review. Hastily slicing a function in half because a tool told someone to do so is likely to lead to an even worse outcome than leaving it intact for reviewers to comment on.

Reply:
I disagree that breaking down a complex function "hastily" will normally be a bad thing.

Reply:
Maybe making this less restrictive than 15 statements would be better. There are some cases where just setting up variable declarations counts as "statements" when there's no way to avoid doing that, and forcing a variation in style to just bunch things with commas instead of declaring each variable separately.  If there was a way to define a statement as something that actually performed an operation, then it would be better, but if not, just making this less restrictive (maybe 20 lines) would help in most cases I've run into.

Reply:
I am good with that as a compromise.